packages/@sanity/form-builder/src/inputs/arrays/ArrayOfObjectsInput/InvalidItem.tsx

```diff
@@ -1,25 +1,28 @@
-import React from 'react'
-import {ArraySchemaType} from '@sanity/types'
-import PatchEvent from '../../../PatchEvent'
-import {resolveTypeName} from '../../../utils/resolveTypeName'
-import InvalidValueInput from '../../InvalidValueInput'
-import {ArrayMember} from './types'
+// @todo: remove this file (unused)
```

Comment:
What's holding us back from removing it now? Perhaps add to the comment why we need to keep it.